components/PostCardList/PostCardListItem.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from "react";
+import styled from "styled-components";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import Post from "../../models/Post";
+import calculateFinalPot from "../../utilities/calculateFinalPot";
+import getAgoByDate from "../../utilities/getRelativeDateString";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+import getSIMetricPrefixData from "../../utilities/getSIMetricPrefixData";
+import Card from "../Card";
+import { ThumbsUpIcon } from "../Icon";
+import PlayingCard from "../PlayingCard";
+import PostTypeContext, { PostType } from "./PostTypeContext";
+
+interface Props extends React.Attributes {
+  post: Post;
+  onClick?: React.MouseEventHandler<HTMLElement>;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItem({ post, ...props }: Props) {
+  const { heroIndex, playerCards } = post.gameSituation;
+  const { left, right } = playerCards[heroIndex]!;
+  const postType = React.useContext(PostTypeContext);
+  const [nAgo, mobileTerm, term] = getAgoByDate(
+    postType === PostType.recent ? post.createdAt : post.lastUpdatedAt
+  );
```

Comment:
This function's return value signature is quite tricky. Can you tell what `nAgo`, `mobileTerm` and `term` is, in an instant in the future?